### ISSUE TITLE
Semi-transparent display of images 

### ISSUE THREAD

**Issue by Hboni** [brainvisa/casa-distro#100]
I'm using casa-distro with ubuntu-18.04, with brainvisa compiled in python3 Qt5.

When I try to open an image in anatomist, it appears semi-transparent. It also happens in some QC visualization with integrated anatomist windows.

![Capture du 2020-07-31 15-09-37](https://user-images.githubusercontent.com/27290853/89039769-eb777c00-d342-11ea-9e18-df5d2d39e76a.png)

When I load an image : 
```
all python modules loaded
Multitexturing present
function glActiveTexture found.
function glClientActiveTexture found.
function glBlendEquation found.
function glTexImage3D found.
function glMultiTexCoord3f found.
function glBindFramebuffer found.
function glBindRenderbuffer found.
function glFramebufferTexture2D found.
function glGenFramebuffers found.
function glGenRenderbuffers found.
function glFramebufferRenderbuffer found.
function glRenderbufferStorage found.
function glCheckFramebufferStatus found.
function glDeleteRenderbuffers found.
function glDeleteFramebuffers found.
Number of texture units: 8
function glUniform1f found.
function glUniform1i found.
function glUniform4fv found.
function glGetUniformLocation found.
function glMultTransposeMatrixf found.
function glAttachShader found.
function glDetachShader found.
function glCompileShader found.
function glCreateProgram found.
function glCreateShader found.
function glDeleteProgram found.
function glDeleteShader found.
function glGetProgramiv found.
function glGetShaderiv found.
function glLinkProgram found.
function glShaderSource found.
function glUseProgram found.
GL_ARB_shadow present
GL_SGIX_shadow extension not available
GL_SGIX_depth_texture extension not available
GL_ARB_depth_texture extension present
GL_ARB_texture_cube_map extension present
GL_EXT_texture_cube_map extension present
Number of texture units: 8
```

**Comment by ylep** [comment]
Duplicate of https://github.com/brainvisa/casa-distro/issues/59

**Comment by ylep** [comment]
This issue should be fixed in casa-distro 3.0 (upcoming Brainvisa 5.0 release) because we are not shipping the old software-only libGL anymore.
